psalm.xml

```diff
@@ -11,9 +11,253 @@
+        <file name="src/Psl/Str/Byte/strip_suffix.php" />
+        <file name="src/Psl/Str/Byte/trim.php" />
+        <file name="src/Psl/Str/Byte/trim_left.php" />
+        <file name="src/Psl/Str/Byte/trim_right.php" />
+        <file name="src/Psl/Str/Byte/uppercase.php" />
+        <file name="src/Psl/Str/Byte/words.php" />
+        <file name="src/Psl/Str/Byte/wrap.php" />
+        <file name="src/Psl/Str/capitalize.php" />
+        <file name="src/Psl/Str/capitalize_words.php" />
+        <file name="src/Psl/Str/chr.php" />
+        <file name="src/Psl/Str/chunk.php" />
+        <file name="src/Psl/Str/concat.php" />
+        <file name="src/Psl/Str/contains.php" />
+        <file name="src/Psl/Str/contains_ci.php" />
+        <file name="src/Psl/Str/encoding.php" />
+        <file name="src/Psl/Str/ends_with.php" />
+        <file name="src/Psl/Str/ends_with_ci.php" />
+        <file name="src/Psl/Str/fold.php" />
+        <file name="src/Psl/Str/format.php" />
+        <file name="src/Psl/Str/format_number.php" />
+        <file name="src/Psl/Str/from_code_points.php" />
+        <file name="src/Psl/Str/is_empty.php" />
+        <file name="src/Psl/Str/is_string.php" />
+        <file name="src/Psl/Str/join.php" />
+        <file name="src/Psl/Str/length.php" />
+        <file name="src/Psl/Str/levenshtein.php" />
+        <file name="src/Psl/Str/lowercase.php" />
+        <file name="src/Psl/Str/metaphone.php" />
+        <file name="src/Psl/Str/ord.php" />
+        <file name="src/Psl/Str/pad_left.php" />
+        <file name="src/Psl/Str/pad_right.php" />
+        <file name="src/Psl/Str/repeat.php" />
+        <file name="src/Psl/Str/replace.php" />
+        <file name="src/Psl/Str/replace_ci.php" />
+        <file name="src/Psl/Str/replace_every.php" />
+        <file name="src/Psl/Str/replace_every_ci.php" />
+        <file name="src/Psl/Str/search.php" />
+        <file name="src/Psl/Str/search_ci.php" />
+        <file name="src/Psl/Str/search_last.php" />
+        <file name="src/Psl/Str/search_last_ci.php" />
+        <file name="src/Psl/Str/slice.php" />
+        <file name="src/Psl/Str/splice.php" />
+        <file name="src/Psl/Str/split.php" />
+        <file name="src/Psl/Str/starts_with.php" />
+        <file name="src/Psl/Str/starts_with_ci.php" />
+        <file name="src/Psl/Str/strip_prefix.php" />
+        <file name="src/Psl/Str/strip_suffix.php" />
+        <file name="src/Psl/Str/to_int.php" />
+        <file name="src/Psl/Str/trim.php" />
+        <file name="src/Psl/Str/trim_left.php" />
+        <file name="src/Psl/Str/trim_right.php" />
+        <file name="src/Psl/Str/truncate.php" />
+        <file name="src/Psl/Str/uppercase.php" />
+        <file name="src/Psl/Str/width.php" />
+        <file name="src/Psl/Str/wrap.php" />
+        <file name="src/Psl/invariant.php" />
+        <file name="src/Psl/invariant_violation.php" />
+        <file name="src/Psl/sequence.php" />
```

Comment:
@muglug is there a way to specify directory or pattern instead? this is gonna be hard to maintain in the future :/ 

related to https://github.com/vimeo/psalm/issues/2861